realsense_ros_camera/launch/rs_camera.launch

```diff
@@ -34,8 +34,10 @@
   <arg name="enable_pointcloud"   default="false"/>
   <arg name="enable_sync"         default="false"/>
 
+  <arg name="base_frame_id"       default="realsense"/>
+  <arg name="group_namespace"       default="realsense"/>
 
-  <group ns="camera">
+  <group ns="$(arg group_namespace)">
     <include file="$(find realsense_ros_camera)/launch/includes/nodelet.launch.xml">
```

Comment:
I'd do this instead:

1. Use `pass_all_args="true"` (see http://wiki.ros.org/roslaunch/XML/include), so you have:
`<include file="$(find realsense_ros_camera)/launch/includes/nodelet.launch.xml" pass_all_args="true"/>`
2. In order to do the suggestion above, the `group_namespace` arg should be passed/taken on the `nodelet.launch.xml` file. Therefore, here you only have the include w/o any `group`. Not sure if that it's fine with other uses of `nodelet.launch.xml` though, but if possible, I'd do that.

I'd also rename `group_namespace` into just `namespace`.